packages/neuron-wallet/src/controllers/update.ts

```diff
@@ -76,11 +77,18 @@ export default class UpdateController {
       if (UpdateController.isChecking) {
         UpdateController.isChecking = false
         UpdateController.updatePackageSize = info.files[0].size ?? 0
-        this.notify({
-          version: info.version,
-          releaseDate: info.releaseDate,
-          releaseNotes: info.releaseNotes as string,
+        const request = net.request(`https://api.github.com/repos/nervosnetwork/neuron/releases/tags/v${info.version}`)
```

Comment:
Added a new branch `request.on('error', () => {` for handling error.  Additionally if the `published_at` field cannot be retrieved, the page will not display the release date to avoid time inconsistencies.
<img width="852" alt="image" src="https://github.com/nervosnetwork/neuron/assets/11756869/d661e9e4-372e-4da6-a204-3ecf134d5014">